### PR TITLE
Reception checker

### PR DESCRIPTION
**Pull request by Luleherll**
This will merge registration_checker branch which contains a program that checks if a guest is registered to develop branch

### CODE REVIEW

**Comment by ekumamait** [comment]
Nice work just add the .vscode and pipfile in a .gitignore otherwise 👍🏾

File: Registration_checker/main.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    first_name = input("Enter your first name:")
+    your_name = str(first_name.capitalize())
+    get_name = ''
+    all_guests = reception.guest_list('ordinary_list.txt','vip_list.txt')
+    
+    for name in all_guests[0]:
+        get_name = name.split(' ', 1)[0]
+        if get_name == your_name:
+            return name + ', VIP'
+    else:
```
Comment (kaggwachristopher) [inline Registration_checker/main.py]: I dont think this indentation is right

File: Registration_checker/main.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        for name in all_guests[1]:
+            get_name = name.split(' ', 1)[0]
+            if get_name == your_name:
+                return name + ', ORDINARY'
+
```
Comment (kaggwachristopher) [inline Registration_checker/main.py]: its always recommended to put an else statement after every if statement and rather put a pass

File: Registration_checker/main.py (hunk 1 above)
Comment (Luleherll) [reply Registration_checker/main.py]: its right. the else is part of a for loop

File: Registration_checker/main.py (hunk 2 above)
Comment (Luleherll) [reply Registration_checker/main.py]: thanks, am going to that

**Pull request merged**